Makefile

```diff
@@ -1,70 +1,92 @@
-SHELL := /usr/bin/env bash
+SHELL := /bin/bash
 
-TEMPLATEFILES := $(shell find ./templates -type f -name "*.yaml.tmpl")
+_PWD := $(shell pwd -P)
 
-BASE_IMG=managed-cluster-validating-webhooks
-NAMESPACE ?= openshift-validation-webhook
-SVCNAME ?= validation-webhook
-SANAME ?= validation-webhook
-GIT_HASH=$(shell git rev-parse --short=7 HEAD)
-IMAGETAG=${GIT_HASH}
-CABUNDLECONFIGMAP ?= webhook-cert
-VWC_ANNOTATION ?= managed.openshift.io/inject-cabundle-from
+GIT_HASH := $(shell git rev-parse --short=7 HEAD)
+IMAGETAG ?= ${GIT_HASH}
 
+BASE_IMG ?= managed-cluster-validating-webhooks
 IMG ?= quay.io/app-sre/${BASE_IMG}
 
-SELECTOR_SYNC_SET_TEMPLATE_DIR=deploy
-BUILD_DIRECTORY=build
+BINARY_FILE ?= build/_output/webhooks
+INJECTOR_BIN ?= build/_output/injector
+
+GO_SOURCES := $(find $(_PWD) -type f -name "*.go" -print)
```

Comment:
Can you show what that would look like? My understanding of `go list ./...` is that it lists packages, and not files. The purpose of this line is to set up dependencies within the Makefile for various targets, and `go list ./...` won't facilitate it?

Reply:
Oh, I see. This file is pretty different than our normal boilerplate. I'm not sure that running a find every command is really worth it as far as the dependency resolution, but 🤷🏻‍♂️

Also, it might be preferable to use `$(CURDIR)` instead of PWD, so that you get the current directory the Makefile itself is in, rather than the working directory of bash.

Reply:
Verified `$(CURDIR)` plays nice with symlinks and updated!